cmsdb/processes/ewk.py

```diff
@@ -393,151 +507,85 @@
     },
 )
 
+
+# LO cross section, needed for scaling to NNLO:
+# based on GenXSecAnalyzer
+# for WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 (Summer20UL16, LO)
+# using command ./calculateXSectionAndFilterEfficiency.sh -f datasets.txt -c RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1 -n 5000000  # noqa
+
+w_lnu_lo_13TeV_xsec = Number(54070.0, {"tot": 18.32})
+
 # LO cross sections, scaled to NNLO
-# inclusive cross section based on WJetsToLNu_TuneCP5_13TeV-madgraphMLM-pythia8 (Summer20UL16, LO)
-# ht bins based on datasets WJetsToLNu_HT-{i}To{j}_TuneCP5_13TeV-madgraphMLM-pythia8 (Summer20UL16, LO)
-w_lnu_ht70To100 = w_lnu.add_process(
-    name="w_lnu_ht70To100",
+
+# ht bins based on GenXSecAnalyzer
+# for WJetsToLNu_HT-{i}To{j}_TuneCP5_13TeV-madgraphMLM-pythia8 (Summer20UL16, LO)
+# using command ./calculateXSectionAndFilterEfficiency.sh -f datasets.txt -c RunIISummer20UL16MiniAODv2-106X_mcRun2_asymptotic_v17-v1 -n 5000000  # noqa
+w_lnu_ht70to100 = w_lnu.add_process(
+    name="w_lnu_ht70to100",
     id=6110,
-    xsecs={13: w_lnu.get_xsec(13) * 1264.0 / 53870.0},
```

Comment:
The previous cross sections were provided from the xsdb: https://xsdb-temp.app.cern.ch
Is there a reason/advantage to use the cross sections from the GenXSecAnalyzer?

Reply:
It is the same method of calculation as the entries on xsdb (just getting the xsec value used for generating the miniaods thanks to the tool), only that 1) we know exactly what parameters were used for the calculation (e.g. how many events we average on, which cmssw version was used...), 2) we are getting the corresponding logs and 3) we are probably more precise as we are using 5M events when possible which seems to give much more precise results than what is seen in xsdb (probably mostly done with the default number of events: 1M)